app/src/models/uncommitted-changes-strategy.ts

```diff
@@ -0,0 +1,5 @@
+export enum UncommittedChangesStrategy {
+  askForConfirmation,
```

Comment:
is this the state we use when the popup is first opened and we don't have an answer from the user yet?

Reply:
This enum controls what to do about uncommitted changes:
* `askForConfirmation` - informs what to do about changes
* `stashOnCurrentBranch`/`moveToNewBranch` - informs what action to take with changes